frontend/app/src/screens/diff/diff-tree.tsx

```diff
@@ -92,7 +94,7 @@ export default function DiffTree({ nodes, loading, ...props }: DiffTreeProps) {
     setTreeData(addItemsToTree(EMPTY_TREE, [...parents, ...formattedNodes]));
   }, [nodes.length]);
 
-  if (treeData.length <= 1) return null;
+  if (treeData.length <= 1) return <NoDataFound message={emptyMessage ?? "No data to display."} />;
```

Comment:
Instead of displaying a error when a given filter is empty, imo it would be better to just disable the filter button

Reply:
done